rubin_sim/moving_objects/make_lsst_obs.py

```diff
@@ -209,34 +216,69 @@ def make_lsst_obs():
         logging.basicConfig(level=logging.INFO)
 
     # Read orbits.
-    orbits = mo.read_orbits(args.orbit_file)
+    orbits = mo.Orbits()
+    orbits.read_orbits(args.orbit_file)
 
+    if args.positions_file is not None:
+        position_data = np.load(args.positions_file)
+        object_positions = position_data["positions"].copy()
+        object_mjds = position_data["mjds"].copy()
+        position_data.close()
+    else:
+        object_positions = None
+        object_mjds = None
     # Read pointing data
     colmap = col_map_dict("fbs")
     pointing_data = mo.read_observations(
         args.simulation_db,
         colmap,
         constraint=args.sql_constraint,
-        footprint=args.footprint,
         dbcols=None,
     )
-    # Generate ephemerides.
-    mo.run_obs(
+
+    d_obs = mo.DirectObs(
```

Comment:
Since this is hard-coded here, it makes sense to remove the option to use LinearObs in the arguments. 
I mentioned earlier removing LinearObs as a class, but I think you could leave it -- but it should match the API for DirectObs, so would need an update so that it has the same API. (the point being that someone else could make a different method for calculating the observations .. such as using SpicePy or whatever .. and then just drop it in in place of DirectObs, only rewriting the "run" method).

Reply:
LinearObs was removed.